### PR TITLE
Feature/handle drive specific files

### PR DESCRIPTION
**Pull request by ronjagrosz**
Added files that are drive specific like drive document or spreadsheet will be clickable.
When clicked the edit page in drive for the file opens.
If it is not a drive specific file (but from drive), it gets downloaded.

### CODE REVIEW

File: app/assets/templates/file_list.html [hunk 1]
```diff
@@ -21,7 +21,10 @@
             <a ng-if="file.filetype === 'bruse/url'" ng-href="{{file.url}}" target="_blank">
               {{file.name}}
             </a>
-            <span ng-if="file.filetype != 'bruse/url'" ng-click="download(file.identity.id, file)">
+            <a ng-if="file.filetype.indexOf('google-apps') > -1" ng-href="{{file.link}}" target="_blank">
+              {{file.name}}
+            </a>
+            <span ng-if="file.filetype != 'bruse/url' && file.filetype.indexOf('google-apps') < 0" ng-click="download(file.identity.id, file)">
```
Comment (ronjagrosz) [inline app/assets/templates/file_list.html:27]: Is it ok to have a long if? I tried a switch to make the code shorter, but it didn't work as I wanted it to.

File: db/schema.rb [hunk 2]
```diff
@@ -80,6 +81,7 @@
     t.datetime "confirmation_sent_at"
     t.string   "unconfirmed_email"
     t.boolean  "own_password",           default: true
+    t.integer  "default_identity_id"
```
Comment (ronjagrosz) [inline db/schema.rb:84]: Was this added by another feature? Because I didn't add it. :confused:

Comment (danielronnkvist) [inline db/schema.rb:84]: Yes, in pull request #22.

File: app/assets/templates/file_list.html (hunk 1 above)
Comment (danielronnkvist) [inline app/assets/templates/file_list.html:27]: I think it's ok, if it has to be repeated at several occasions or if more validations has to be made it's better to move it into a function.

**Comment by danielronnkvist** [comment]
Looks and works great! :+1:

**Comment by klaseskilson** [comment]
Looks good! :cool:

**Pull request merged**